frontend-react/src/components/SideNav.tsx

```diff
@@ -0,0 +1,57 @@
+import { SideNav } from "@trussworks/react-uswds";
+import { useLocation } from "react-router-dom";
+
+import { USNavLink } from "./USLink";
+
+export interface SideNavItemProps
+    extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+    items?: React.ReactNode[];
+    children: React.ReactNode;
+    to?:
+        | string
+        | React.FunctionComponent<
+              React.AnchorHTMLAttributes<HTMLAnchorElement> & {
+                  children: React.ReactNode;
+              }
+          >;
+    isActive?: boolean;
+}
+
+/**
+ * Wrapper component that provides the NavLink and SideNav sibling elements (if items
+ * provided). The parent navlink can be customized by passing a functional component
+ * or html element string as the "to" prop (defaults to USNavLink). The SideNav sibling
+ * can be forced visible via the "isActive" prop (otherwise defaults to checking path).
+ */
+export function SideNavItem({
+    href,
+    children,
+    items,
+    to: NavLink = USNavLink,
+    isActive,
+    ...props
+}: SideNavItemProps) {
+    const { pathname } = useLocation();
+    const isSubnavVisible =
+        isActive !== undefined
+            ? isActive
+            : href === undefined ||
+              href === "" ||
+              pathname === href ||
+              pathname.startsWith(href);
+    const subnavClassname = !isSubnavVisible ? "display-none" : "";
+
+    // SideNav doesn't allow custom classes so we have to wrap in a div 😡
```

Comment:
Honestly, I don't think it's a huuuuuge deal if we just want to build our own since [their SideNav](https://github.com/trussworks/react-uswds/blob/main/src/components/SideNav/SideNav.tsx) is so simplistic anyway.  Let's hold off on that and reconsider if it becomes an issue going forward, though!